### DISCUSSION TITLE
Having a problem, lightspark isn't starting

### DISCUSSION THREAD

**Discussion by Creatingthisaccounttofixlightspark** [lightspark/lightspark#952] [Q&A] [answered]
When i do "lightspark game.swf"

it writes :

"INFO: Lightspark version 0.8.5 Copyright 2009-2013 Alessandro Pignotti and others
INFO: RenderThread this=0x564604cebc00
INFO: Creating input thread
INFO: Running in local-with-filesystem sandbox
INFO: zlib compressed SWF file: Version 9
INFO: FrameRate 24
INFO: Creating VM
INFO: Switched to local-with-networking sandbox by FileAttributesTag
ERROR: Error while reading tag 75. Size=14 expected: 16
INFO: Fragment shader compilation 
INFO: Vertex shader compilation 
INFO: throwing exception:0x564605bb6900 id:1074 Error #1074: Illegal write to read-only property width on Class.
terminate called after throwing an instance of 'lightspark::ReferenceError*'
Abandon"

I have tried to remake the previous commands, but nothing changed.

Can someone help me please

Also i'm on Ubuntu 20.04

**Comment by eiglow** [comment]
Can you upload your game.swf so that we test and debug it?

**Reply by Creatingthisaccounttofixlightspark** [reply]
Sure ! Also thanks you for trying to help me.

Where i get it :

https://archive.org/details/game_20210129_20210129_1811

The file :

https://archive.org/download/game_20210129_20210129_1811/game.swf

**Comment by dbluelle** [comment, marked as answer]
The issue is now fixed (see https://github.com/lightspark/lightspark/commit/b652cdc0a4dec9ba39218c58d7b2b707775dd7a6)
